tests/e2e/specs/main/gps.test.ts

```diff
@@ -7,8 +7,8 @@ describe('MAIN - GPS Component Test', () => {
     const mapView = await $(byResourceId('MAIN.mapbox-map-view'));
     await expect(mapView).toBeDisplayed();
 
-    const gpsText = await $(byResourceId('MAP.gps-pill-text'));
-    await expect(gpsText).toBeDisplayed();
+    const gpsPill = await $(byResourceId('MAP.gps-pill'));
+    await expect(gpsPill).toBeDisplayed();
```

Comment:
I removed the test ID that was used internally by this component and updated this test to confirm that the pill as a whole is displayed, not the text component within the pill. i think it's a minor change but maybe that's not desired.